### PR TITLE
feat(loot): add drop rarity embed field

### PR DESCRIPTION
**Pull request by iProdigy**
Work towards #139

- [x] Load NPC drop rates
- [x] Include drop rarity as embed field
- [ ] Load pickpocket drop rates
- [ ] Load "event" drop rates
- [ ] Clue notifier integration (possibly for another PR)
- [ ] Collection notifier integration (likely for another PR)
- [ ] Min Rarity config (likely for another PR)

image

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -3,6 +3,9 @@ BSD 2-Clause License
 Copyright (c) 2022, Jake Barter
 All rights reserved.
 
+Copyright (c) 2020, MasterKenth
+All rights reserved.
```
Comment (pajlada) [inline LICENSE:7]: Thoughts on including reason why something is included in the license?
```suggestion
Copyright (c) 2020, MasterKenth
All rights reserved.
Drop rate information from the Discord Rare Drop Notifier plugin
```

Comment (iProdigy) [reply LICENSE:7]: I'm actually in the process of freshly generating the json from wiki parsing via python scripts, so we'll have the latest drop rates (& not need to update our license)

Comment (pajlada) [reply LICENSE:7]: Nice! Can be used with a github actions cronjob to keep us up to date too

Comment (iProdigy) [reply LICENSE]: Updated in https://github.com/pajlads/DinkPlugin/pull/141/commits/88331ec8f35cface89444d6c4575a0e5127c439e - will do scripts+cronjob in another pr

**Comment by iProdigy** [comment]
For context, PR is in draft stage until https://github.com/osrsbox/osrsbox-db is updated to correctly parse latest drop data (or we write the scripts do this parsing ourselves). Note: even more active forks of osrsbox like https://github.com/0xNeffarion/osrsreboxed-db don't have full coverage of relevant wiki pages

**Comment by iProdigy** [comment]
Obtained a more accurate data source, will create another PR

**Pull request closed without merging**